eq-author/src/components/QuestionnaireTable/Row.js

```diff
@@ -258,6 +261,118 @@ export const Row = ({
   const hasWritePermission = permission === WRITE;
   const canDelete = hasWritePermission && !locked;
 
+  const renderEnabled = (tableHeadings, columnHeading) =>
+    tableHeadings.map(({ heading, enabled }) => {
+      if (enabled) {
+        if (heading === columnHeading) {
+          if (heading === Headings.TITLE) {
```

Comment:
As each if statement returns, you could swap the series of `if` statements to be a `switch` block.

```
switch (heading) {
   case Headings.TITLE:
      . . .
     break;
   case Headings.OWNER:
      . . .
     break;
   case Headings.CREATED:
     . . .
     break;
   . . .
   default:
     return null;
}
```